src/precice/impl/Participant.hpp

```diff
@@ -73,98 +72,178 @@ class Participant {
       const mesh::PtrData &data,
       const mesh::PtrMesh &mesh);
 
+  void addReadMappingContext(MappingContext *mappingContext);
+
+  void addWriteMappingContext(MappingContext *mappingContext);
+
+  void addWatchPoint(const PtrWatchPoint &watchPoint);
+
+  void addWatchIntegral(const PtrWatchIntegral &watchIntegral);
+
+  void setUseMaster(bool useMaster);
+
+  void setMeshIdManager(std::unique_ptr<utils::ManageUniqueIDs> &&idm)
+  {
+    _meshIdManager = std::move(idm);
+  }
+
+  void addAction(action::PtrAction &&action);
```

Comment:
Could you add a short description of what these function do, even when it seems obvious to you?